src/lib/apis/models/index.ts

```diff
@@ -63,10 +63,7 @@ export const getModelInfos = async (token: string = '') => {
 export const getModelById = async (token: string, id: string) => {
 	let error = null;
 
-	const searchParams = new URLSearchParams();
```

Comment:
Mistake here? I don't see why this was changed?

Reply:
Yes you are right. The change was based on a integration test in backend which was not able to get the model by id. Looked like a conflict in the router with same path but different query parameters.
Reverted the change